internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -7684,6 +7706,148 @@ func TestHandleGetRawTransaction(t *testing.T) {
 	}})
 }
 
+func TestHandleVersion(t *testing.T) {
+	t.Parallel()
+
+	runtimeVer := strings.Replace(runtime.Version(), ".", "-", -1)
+	version.BuildMetadata = "foo"
+	version.PreRelease = "pre"
+	buildMeta := version.NormalizeBuildString(
+		fmt.Sprintf("%s.%s", version.BuildMetadata, runtimeVer))
+
+	result := map[string]types.VersionResult{
+		"dcrdjsonrpcapi": {
+			VersionString: "7.0.0",
+			Major:         7,
+			Minor:         0,
+			Patch:         0,
+			Prerelease:    "",
+			BuildMetadata: "",
```

Comment:
This test will fail (and need to be updated) whenever the API version changes.  You may want to replace these with `jsonrpcSemverString`, etc., to avoid that.  Though at that point, it's worth pointing out that this function is almost identical to the actual function being tested, so it probably has very marginal value.